src/server/system_services/bucket_server.js

```diff
@@ -384,6 +384,41 @@ async function get_bucket_policy(req) {
     };
 }
 
+async function get_bucket_policy_status(req) {
+    dbg.log0('get_bucket_policy_status:', req.rpc_params);
+    const bucket = find_bucket(req, req.rpc_params.name);
+    const policy = bucket.s3_policy;
+    if (!policy) return {};
+    const isPublic = _is_policy_public(policy);
```

Comment:
for internal variables please use our format of is_public
```suggestion
    const is_public = _is_policy_public(policy);
```